catppuccin.kdl

```diff
@@ -1,60 +1,57 @@
-// Catppuccin Theme:
-// https://github.com/catppuccin/catppuccin
-
 themes {
   catppuccin-latte {
-    bg "#acb0be" // Surface2
-    fg "#acb0be" // Surface2
```

Comment:
These were both originally `surface2` just on latte? Doesn't make sense to have the same `bg` and `fg` so I'm not going to add a latte-only change here I think.